packages/web-react/src/components/Toast/useToastStyleProps.ts

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+import { SpiritToastProps } from '../../types';
+
+export interface ToastStyles<T> {
+  classProps: {
+    root: string;
+    queue: string;
+  };
+  props: T;
+}
+
+export function useToastStyleProps(props: SpiritToastProps): ToastStyles<SpiritToastProps> {
+  const { alignmentX, alignmentY, ...restProps } = props;
+
+  const toastClass = useClassNamePrefix('Toast');
+  const classes: string[] = [];
+
+  function processAlignment(alignment: SpiritToastProps['alignmentX'] | SpiritToastProps['alignmentY']) {
+    if (typeof alignment === 'object' && alignment !== null) {
+      Object.keys(alignment).forEach((key) => {
+        const infix = key === 'mobile' ? '' : `--${key}`;
+        classes.push(`${toastClass}${infix}--${alignment[key as keyof typeof alignment]}`);
+      });
+    } else {
+      classes.push(`${toastClass}--${alignment}`);
+    }
+  }
+
+  processAlignment(alignmentX);
```

Comment:
I think you can process both alignments in one function using `['alignmentX', 'alignmentY'].forEach(...)`.

Reply:
joined to one function